src/main/resources/dockerfiles/Dockerfile-skeleton-7

```diff
@@ -0,0 +1,92 @@
+ARG BASE_IMAGE=docker.io/xenit/tomcat:alfresco-6.0-ubuntu
+FROM $BASE_IMAGE
+
+# ${CATALINA_HOME} is set to ${CATALINA_HOME} in upstream
+
+USER	root
+
+# downloading ALF & components
+RUN	mkdir -p /opt/alfresco && \
+	apt-get update && \
+	apt-get install -y curl unzip vim locate xmlstarlet jq gettext-base fontconfig && \
+	rm -rf ${CATALINA_HOME}/webapps/* && \
+	mkdir -p ${CATALINA_HOME}/bin && \
+	touch ${CATALINA_HOME}/bin/setenv.sh && \
+    #
+    # Configure server.xml
+    xmlstarlet edit --inplace \
+        --append "/Server/Service/Connector" --type elem --name Connector \
+            --var connector '$prev' \
+            --insert '$connector' --type attr --name port --value \$\{TOMCAT_PORT_SSL\} \
+            --insert '$connector' --type attr --name URIEncoding --value UTF-8 \
+            --insert '$connector' --type attr --name protocol --value org.apache.coyote.http11.Http11NioProtocol \
```

Comment:
I had to change this protocol to Http11NioProtocol because Http11Protocol was removed from Tomcat 9